source/includes/read/retrieve.rb

```diff
@@ -0,0 +1,47 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+begin
+  client = Mongo::Client.new(uri)
+
+  # start-db-coll
+  database = client.use('sample_training')
+  collection = database[:companies]
+  # end-db-coll
+
+  # Finds one document with a "name" value of "LinkedIn"
+  # start-find-one
+  document = collection.find({ 'name' => 'LinkedIn' }).first
+  puts document
```

Comment:
Q: When are/aren't we using parentheses for method uses?

Reply:
The current Ruby docs don't use parentheses next to the method name. It seems like you can leave out empty parentheses when coding in Ruby, so I think it follows that convention